engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -231,22 +277,35 @@ static PDType1Font getStandardFont(String name)
         }
         //duplicating until here
 
+        private String fontName = null;
+        private boolean fontChanged = false;

```

Comment:
I don't think this is used.

Reply:
You're correct, the `fontChanged`variable is a leftover from previous tests that have been removed.